core/federated/RTI/rti_remote.c

```diff
@@ -2330,6 +2332,7 @@ void* lf_delayed_grants_thread(void* nothing) {
           free(next);
         }
       }
+      LF_MUTEX_UNLOCK(&rti_mutex);
```

Comment:
This unlock may occur with no matching lock.

Reply:
This is also outdated and the problem is solved.